### PR TITLE
MM-16993 - Add status messages for create/deploy/destroy events

### PR DESCRIPTION
**Pull request by reflog**

### CODE REVIEW

File: cmd/ltops/destroy.go [hunk 1]
```diff
@@ -23,8 +25,11 @@ var destroyCluster = &cobra.Command{
 		if err != nil {
 			return errors.Wrap(err, "Couldn't load cluster")
 		}
-
-		return cluster.Destroy()
+		mlog.Info("Destroying cluster...")
+		if err = cluster.Destroy(); err == nil {
```
Comment (crspeller) [inline cmd/ltops/destroy.go]: Can we switch this around to be idiomatic?

**Review by crspeller — CHANGES_REQUESTED** [review]

File: cmd/ltops/destroy.go (hunk 1 above)
Comment (reflog) [reply cmd/ltops/destroy.go]: @crspeller - do you mean turn it into 'err != nil' ?

Comment (crspeller) [reply cmd/ltops/destroy.go]: Yes. Exactly.

File: cmd/ltops/destroy.go [hunk 2]
```diff
@@ -23,8 +25,11 @@ var destroyCluster = &cobra.Command{
...
+			mlog.Info("Custer destroyed successfully")
```
Comment (lieut-data) [inline cmd/ltops/destroy.go]:
```suggestion
			mlog.Info("Cluster destroyed successfully")
```

**Review by lieut-data — CHANGES_REQUESTED** [review]
Just one typo below.

**Review by lieut-data — APPROVED** [review]

**Review by crspeller — APPROVED** [review]

**Pull request merged**